src/doc/src/reference/unstable.md

```diff
@@ -1230,66 +1230,134 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
+This would be in [`workspaces.md`][workspaces], under
+[The `workspace.metadata` table][the-workspacemetadata-table]
 
-```toml
-# in workspace's Cargo.toml
-[workspace.dependencies]
-log = "0.3.1"
-log2 = { version = "2.0.0", package = "log" }
-serde = { git = 'https://github.com/serde-rs/serde' }
-wasm-bindgen-cli = { path = "crates/cli" }
-```
+The `workspace.package` table is where you define keys that can be
+inherited by members of a workspace. These keys can be inherited by
+defining them in the member package with `{key}.workspace = true`.
+For Example:
 
 ```toml
-# in a workspace member's Cargo.toml
-[dependencies]
-log.workspace = true
-log2.workspace = true
-```
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
 
-Example 2:
-```toml
-# in workspace's Cargo.toml
 [workspace.package]
 version = "1.2.3"
 authors = ["Nice Folks"]
 description = "..."
 documentation = "https://example.github.io/example"
-readme = "README.md"
-homepage = "https://example.com"
-repository = "https://github.com/example/example"
-license = "MIT"
-license-file = "./LICENSE"
-keywords = ["cli"]
-categories = ["development-tools"]
-publish = false
-edition = "2018"
 ```
 
 ```toml
-# in a workspace member's Cargo.toml
+# [PROGECT_DIR]/bar/Cargo.toml
 [package]
+name = "bar"
 version.workspace = true
 authors.workspace = true
 description.workspace = true
 documentation.workspace = true
-readme.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
-license-file.workspace = true
-keywords.workspace = true
-categories.workspace = true
-publish.workspace = true
 ```
 
+Keys that are supported:
+`authors`, `categories`, `description`, `documentation`, `edition`,
+`exclude`, `homepage`, `include`, `keywords`, `license`, `license-file`,
+`publish`, `readme`, `repository`, `rust-version`, and`version`.
+
+
+### The `workspace.dependencies` table
+
+The `workspace.dependencies` table is where you define dependencies to be
+inherited by members of a workspace. Inheriting dependencies was added as a
+way to reduce duplication of dependencies across members of a workspace.
+Inheriting a dependency is as simple as setting the dependency in the workspace
+under `[workspace.dependencies]` as you would in`[dependencies]`. From there
+[specify that dependency][inheriting-a-dependency-from-a-workspace] in the member
+`[dependencies]` table.
+Full example:
+
+```toml
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
+
+[workspace.dependencies]
+log = "0.3.1"
+serde = { git = 'https://github.com/serde-rs/serde' }
+wasm-bindgen-cli = { path = "crates/cli" }
+```
+
+```toml
+# [PROGECT_DIR]/bar/Cargo.toml
+[project]
+name = "bar"
+version = "0.2.0"
+
+[dependencies]
+log.workspace = true
+serde.workspace = true
+wasm-bindgen-cli.workspace = true
+```
+
+Dependencies in `[workspace.dependencies]` cannot be declared as `optional`.
+The `optional` key must be omitted or, if present, must be `false`. Dependencies
```

Comment:
These two sentences feel redundant.  I know there is a slight distinction but we either should collapse them down or decide the distinction isn't worth it and leave off the second

Reply:
Dropping the second one seems better. It is the more restrictive of the two